include/myst/gcov.h

```diff
@@ -4,9 +4,11 @@
 #ifndef _MYST_GCOV_H
 #define _MYST_GCOV_H
 
+#include <errno.h>
 #include <myst/libc.h>
```

Comment:
I think we can remove ``<myst/libc.h>`` now. In fact, we can remove all the headers from this file.

Reply:
Removed them